spec/device_detector/client_fixtures_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../spec_helper'
+
+describe DeviceDetector::Client do
+
+  fixture_dir = File.expand_path('../../fixtures/client', __FILE__)
+  fixture_files = Dir["#{fixture_dir}/*.yml"]
+  fixture_files.each do |fixture_file|
+
+    describe File.basename(fixture_file) do
+
+      fixtures = YAML.load_file(fixture_file)
+      fixtures.each do |f|
+
+        user_agent = f["user_agent"]
+        client = DeviceDetector::Client.new(user_agent)
+
+        describe user_agent do
+
+          it "should be known" do
+            assert client.known?, "isn't known as a client"
```

Comment:
You have been using the `Test::Unit` matchers, which is fine, but the rest of the specs are using `Minitest::Spec`. I already wanted to do the switch, but I think it's better to get this PR reviewed first and open a new one with the change to the test suite.